test/common/testlib.py

```diff
@@ -490,11 +490,11 @@ def _key_press_chromium(self, keys: str, modifiers: int = 0, use_ord=False):
             self.cdp.invoke("Input.dispatchKeyEvent", **args)
 
     def _key_press_firefox(self, keys: str, modifiers: int = 0, use_ord: bool = False):
-        # https://github.com/GoogleChrome/puppeteer/blob/master/lib/USKeyboardLayout.js
+        # https://python-reference.readthedocs.io/en/latest/docs/str/ASCII.html
         keyMap = {
             8: "Backspace",   # Backspace key
             9: "Tab",         # Tab key
-            13: "Enter",      # Enter key
+            10: "Enter",      # Enter key
```

Comment:
13 is carriage return (enter key)
10 is not enter, but linefeed

The issue is the handling of CRLF.

See https://github.com/cockpit-project/cockpit-machines/pull/1150#issuecomment-1748497292